src/python/pants/backend/python/tasks2/python_execution_task_base.py

```diff
@@ -50,25 +50,14 @@ def path(self):
     return self._pex.path()
 
   def cmdline(self, args=()):
-    cmdline = ' '.join(self._pex.cmdline(args))
-    pex_path = self._pex_path()
-    if pex_path:
-      return '{env_var_name}={pex_path} {cmdline}'.format(env_var_name=self._PEX_PATH_ENV_VAR_NAME,
-                                                          pex_path=pex_path,
-                                                          cmdline=cmdline)
-    else:
-      return cmdline
+    return ' '.join(self._pex.cmdline(args))
 
   def run(self, *args, **kwargs):
     kwargs_copy = copy(kwargs)
     env = copy(kwargs_copy.get('env')) if 'env' in kwargs_copy else {}
-    env[self._PEX_PATH_ENV_VAR_NAME] = self._pex_path()
     kwargs_copy['env'] = env
     return self._pex.run(*args, **kwargs_copy)
```

Comment:
the only reason I can see to do all of the kwargs + env copying here would be to mutate that - so I'm thinking that's probably no longer necessary?

i.e. `return self._pex.run(*args, **kwargs)` is probably a sufficient method body.